cmd/cluster-proportional-autoscaler/options/options.go

```diff
@@ -117,7 +123,7 @@ func (c *configMapData) Type() string {
 
 // AddFlags adds flags for a specific AutoScaler to the specified FlagSet
 func (c *AutoScalerConfig) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&c.Target, "target", c.Target, "Target to scale. In format: deployment/*, replicationcontroller/* or replicaset/* (not case sensitive).")
+	fs.StringVar(&c.Targets, "targets", c.Targets, "Targets to scale. In format: 'deployment/*,replicationcontroller/*,replicaset/*' (not case sensitive, comma delimiter supported).")
```

Comment:
You can't break compat without a major change (e.g. 2.0.0).  I suggest you still accept the old flag and add the new flag.  If only the old flag is specified, you can convert it into the new-style list.  If only the new flag is specified, no problem. If both are specified, it can be a fatal error.

Reply:
@thockin Thanks for the comment. 
In that case isn't it better to discard the `targets` flag and just stick with a single flag `target` that will support both single and multiple targets ?  
I know it breaks the meaning of the singular form of the flag `target` because it will support multiple targets too, but it can be stated in the documentation.

Reply:
I agree with just keeping the `target` flag, that was my initial idea.

Reply:
Repurposing the original flag while keeping it backward compatible seems fine to me.